framework/contrib/hit/braceexpr.cc

```diff
@@ -0,0 +1,208 @@
+
+#include "braceexpr.h"
+
+#include <cstdlib>
+#include <iostream>
+
+namespace hit
+{
+
+std::string
+BraceNode::str(int indent)
```

Comment:
This could be const.  It is/was mostly used for debugging brace expression parsing.  Someday, when people start finding bugs in brace expression parsing, there will probably be tests added that use it.